components/link/link.js

```diff
@@ -125,6 +125,7 @@ class Link extends LocalizeCoreElement(FocusMixin(LitElement)) {
 				}
 				d2l-icon.d2l-new-window {
 					color: var(--d2l-color-celestine);
+					margin-inline-start: 0.5ch;
```

Comment:
When I asked Jeff how much space he'd like between the link text and the icon, he said:

```suggestion
					margin-inline-start: 6px;
```

Reply:
Doesn't it seem weird for it to be fixed though?

Reply:
Maybe. This ends up being `5.51px` @ `20px` default font size, so I'm good with it either way.

Reply:
If we make the icon:
```
width: 0.78em;
height: 0.78em;
margin-inline-start: 0.33em; /* 6px default */
```

Then the icon scales with the link no what people do to it.

Reply:
Chatted with Jeff -- he definitely likes the idea of scaling the icon with the text like we do with subtle buttons, so good call.

He'd like for it to be `18px` for "normal" links at default font size, so `0.9em` should achieve that.

For `small` links, they're a bit weird in that the font-size is `rem`-based and not `em`-based, so the font size never actually goes up/down with the cascaded font-size. So for them, I think we want `0.7rem` which would result in `14px`.

Does that make sense?

Reply:
It does except I still think `1em` for the small links makes more sense because it shouldn't really care about anything else at that point except the font-size of the thing it lives inside. So, whether the small link were to change its font-size to `0.8rem`, '16px', or `0.842105263156em`, the icon would still be fine.

Reply:
Ah yep cool cool, carry on 😎

Reply:
Could we just roll with `0.95em` for both then for ultimate consistency and less CSS?